locales/en/exposure-card.ftl

```diff
@@ -0,0 +1,49 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+chevron-down-alt = Expand
+chevron-up-alt = Collapse
+
+exposure-card-company-logo = Company logo
+exposure-card-company = Company domain
```

Comment:
```suggestion
# Domain refers to the company's website
exposure-card-company = Company domain
```

I assume this refers to the website's domain name?

Reply:
I tried to check how it was used, but it looks like this (and `exposure-card-company-logo`) weren't used any more, so I removed them: b3d0d6430